app_dev/android_java.html

```diff
@@ -51,143 +51,149 @@ <h1 class="heading" id="java">Android Development Using Java</h1>
                 title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
                 clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                <h2> youtube video title</h2>
-              <h3> chanel name</h3>
+                <h2>Android App Development Tutorial For Beginners 2021 </h2>
+              <h3>Programming Path</h3>
               <p>
-                Hello this is a video for Web Development purpose. hello this is a video for Web
-                Development purpose. hello this is a video for Web Development purpose
+                Learn how to make online games and Find my phone using Android.And this video was four section  Installing Software,
+                Getting Started with Android App Development, Networking in Android, Build your own Portfolio Android App.
               </p>
             </div>
 
             <div class="video">
                 <iframe width="560" height="315" src="https://www.youtube.com/embed/EplH-amHTtE"
                 title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
                 clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                <h2> youtube video title</h2>
-              <h3> chanel name</h3>
+                <h2>Android App Development Tutorial for Beginners - Hindi</h2>
+              <h3>Neat Roots</h3>
               <p>
-                Hello this is a video for Web Development purpose. hello this is a video for Web
-                Development purpose. hello this is a video for Web Development purpose
+                In this Course, You can learn android app development from Basics to Expert Level from
+                Certified Android Developer. If you don't know any programming language still you can
+                learn everything about android app development.
               </p>
             </div>
 
             <div class="video">
                 <iframe width="560" height="315" src="https://www.youtube.com/embed/O3BVBzV58hg"
                 title="YouTube video player" frameborder="0" allow="accelerometer; autoplay;
                 clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                <h2> youtube video title</h2>
-              <h3> chanel name</h3>
+                <h2>Android Full Course - Learn Android in 14 Hours </h2>
+              <h3>Master Coding</h3>
               <p>
-                Hello this is a video for Web Development purpose. hello this is a video for Web
-                Development purpose. hello this is a video for Web Development purpose
+                In this course, we also learn to solve your code bugs for become play store ready Android
+                Developer Android Development Course. This video is Android Development Tutorial for Beginners -Part 1.
```

Comment:
remove -part 1.

Reply:
`<p>
                In this course, we also learn to solve your code bugs for becoming play store ready Android
                Developer Android Development Course. This video is Android Development Tutorial for Beginners </p>`